tensorflow/core/kernels/eig_op_complex64.cc

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/core/kernels/eig_op_impl.h"
+
+namespace tensorflow {
+
+REGISTER_LINALG_OP("Eig", (ComplexEigOp<complex64, complex64>),
```

Comment:
Now that this supports real type as well, should we simply call the kernel "EigOp" for consistency? EigenDecompositionOp would perhaps be less idiomatic, but more informative. I'm fine with either.